espnet/nets/e2e_asr_th.py

```diff
@@ -2075,14 +2084,14 @@ def forward(self, hs_pad, hlens, ys_pad):
                 logging.info(' scheduled sampling ')
                 z_out = self.output(z_all[-1])
                 z_out = np.argmax(z_out.detach(), axis=1)
```

Comment:
`torch.argmax(z_out.detach(), dim=1)` can be faster because it can use GPU without memory re-transfer by `z_out.cuda()`

Reply:
this is related to https://github.com/espnet/espnet/pull/358#issuecomment-420815239

Reply:
What the performance means in that issue? Is torch.argmax slower than numpy.argmax?